bundles/org.openhab.binding.shelly/src/test/java/org/openhab/binding/shelly/internal/discovery/ShellyThingCreatorTest.java

```diff
@@ -271,7 +287,7 @@ void getThingUIDReturnsThingTypeMatchingServiceName() {
     @ParameterizedTest
     @MethodSource("provideTestCasesForIsValidShellyServiceName")
     void isValidShellyServiceName(String serviceName, boolean expected) {
-        assertThat("serviceName: " + serviceName, ShellyThingCreator.isValidShellyServiceName(serviceName),
+        assertThat("serviceName: " + serviceName, ShellyMDNSDiscoveryParticipant.isValidShellyServiceName(serviceName),
```

Comment:
This testcase should now be moved to a separate test class for `ShellyMDNSDiscoveryParticipant`.

Reply:
done